nodes/hostingerApi/HostingerApi.node.ts

```
@@ -849,6 +1024,23 @@ export class HostingerApi implements INodeType {
 				case 'getPaymentList': method = 'GET'; endpoint = '/api/billing/v1/payment-methods'; break;
 				case 'deleteSubscription': method = 'DELETE'; endpoint = `/api/billing/v1/subscriptions/${getParam('subscriptionId')}`; break;
 				case 'getSubscriptionList': method = 'GET'; endpoint = '/api/billing/v1/subscriptions'; break;
+				//Reach
+				case 'listContacts': 
+					let contactsEndpoint = `/api/reach/v1/contacts?page=${getParam('page')}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)